tirocks-sys/build.rs

```diff
@@ -41,8 +41,10 @@ fn bindgen_rocksdb(file_path: &Path) {
         .allowlist_type(r"\brocksdb::Histograms")
         .allowlist_type(r"\brocksdb::titandb::TickerType")
         .allowlist_type(r"\brocksdb::titandb::HistogramType")
+        .opaque_type(r"\brocksdb::Env")
```

Comment:
why

Reply:
Because we don't care how `Env` is defined and we don't need to import various C++ specific struct into bindings.